pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1001,11 +1001,13 @@ func skipLocalUnderArm64(cloud string) string {
 
 func registerCDC(r registry.Registry) {
 	r.Add(registry.TestSpec{
-		Name:            "cdc/initial-scan-only",
-		Owner:           registry.OwnerCDC,
-		Benchmark:       true,
-		Cluster:         r.MakeClusterSpec(4, spec.CPU(16), spec.Arch(vm.ArchAMD64)),
-		RequiresLicense: true,
+		Name:             "cdc/initial-scan-only",
+		Owner:            registry.OwnerCDC,
+		Benchmark:        true,
+		Cluster:          r.MakeClusterSpec(4, spec.CPU(16), spec.Arch(vm.ArchAMD64)),
+		RequiresLicense:  true,
+		CompatibleClouds: registry.AllExceptAWS,
```

Comment:
I am curious how `AllExceptAWS` was derived? A number of these tests can actually be run in AWS despite the fact that CI currently confines the set, largely to performance tests.

Reply:
Tests that didn't have the `aws` tag.

I did [ask](https://cockroachlabs.slack.com/archives/C023S0V4YEB/p1694786376678939?thread_ts=1694745643.434109&cid=C023S0V4YEB) if we would want any of the tests in the nightly (GCE) suite to not run in AWS nightlies if they can run on AWS, and the answer was no..

If we want a smaller nightly suite for AWS, it will have to be a separate suite. Though I wouldn't make it AWS specific, it could be something like "ReducedNightly".

In any case, when we start using these fields in `roachtest` I will add a flag to ignore cloud compatibility so it will always allow running a test manually if necessary.